test/functional/mempool_timelocked_tx.py

```diff
@@ -81,7 +81,7 @@ def run_test(self):
         node.mempool_submit_transaction(tx1, {})
         node.mempool_submit_transaction(tx3, {})
         # Cannot submit tx2 yet, it spends and unconfirmed time-locked output
-        assert_raises_rpc_error(None, "Timelock rules violated", node.mempool_submit_transaction, tx2, {})
+        assert_raises_rpc_error(None, "locked until", node.mempool_submit_transaction, tx2, {})
```

Comment:
This should be more informative with explicit error information. @azarovh can you please fix this?

Reply:
The information is there. This is just partial string comparison to check that this string is in the error after rpc call.

Reply:
> This should be more informative with explicit error information. @azarovh can you please fix this?

The full error is `"Spending at height {0}, locked until height {1}"`. Just the test matches only a part of it since I am not sure if the heights are stable across runs.